### PR TITLE
gatehost: document deployment

### PR DESCRIPTION
**Pull request by bmwiedemann**
update / document more setup parts of
gatehost1 and gatehost2 Nuremberg infrastructure nodes

### CODE REVIEW

File: hostscripts/gatehost/setup.sh [hunk 1]
```diff
@@ -12,15 +12,17 @@ popd
...
 done
 
 sleep 100 # TODO waitfor ssh
 echo "default root password is linux"
-ssh gatevm mkdir -p ~/.ssh/
-scp ~/.ssh/authorized_keys gatevm:.ssh/
-scp -r /etc/zypp/repos.d gatevm:/etc/zypp/
-ssh gatevm zypper -n in python-pyOpenSSL python-xml # for salt to work
-echo "now use salt-ssh gatevm state.highstate"
+for host in gatevm ; do
```
Comment (nicolasbock) [inline hostscripts/gatehost/setup.sh]: Why is this a loop? Are you planning on adding more hosts here?

Comment (bmwiedemann) [reply hostscripts/gatehost/setup.sh]: yes, we had some more VMs on the same host (e.g. clouddata) that needed pretty much the same code

Comment (nicolasbock) [reply hostscripts/gatehost/setup.sh]: Will you add those in a later PR?

Comment (jdsn) [reply hostscripts/gatehost/setup.sh]: If you plan to add those, please introduce a variable for this now (maybe allow to override from env).

**Comment by jdsn** [comment]
@nicolasbock issue addressed, please rereview

File: hostscripts/gatehost/setup.sh [hunk 2]
```diff
@@ -12,15 +12,18 @@ popd
...
 done
 
 sleep 100 # TODO waitfor ssh
 echo "default root password is linux"
-ssh gatevm mkdir -p ~/.ssh/
-scp ~/.ssh/authorized_keys gatevm:.ssh/
-scp -r /etc/zypp/repos.d gatevm:/etc/zypp/
-ssh gatevm zypper -n in python-pyOpenSSL python-xml # for salt to work
-echo "now use salt-ssh gatevm state.highstate"
+sshhosts="gatevm"
```
Comment (nicolasbock) [inline hostscripts/gatehost/setup.sh]: Should we make this overridable from the ENV (as per @jdsn 's suggestion)?

Comment (jdsn) [reply hostscripts/gatehost/setup.sh]: that was just an idea, but currently it is not needed, so can be done later

**Pull request merged**